export.py

```diff
@@ -158,6 +223,16 @@ def save_as_markdown(id, subfolder, json_post, post_comments_html):
             f.write(post_comments_html)
 
 
+def save_together_as_markdown(id, subfolder, json_post, post_comments_html):
+    subfolder = json_post['eventtime'][:4] # Note's year
+    note_name = json_post['eventtime'][:10] # Note's file name = YYYY-MM-DD
+    os.makedirs('posts-and-comments-markdown/{0}'.format(subfolder), exist_ok=True)
+    with open('posts-and-comments-markdown/{0}/{1}.md'.format(subfolder, note_name), 'a', encoding='utf-8') as f:
+        f.write(json_to_alt_markdown(json_post) + '\n\n')
+        if post_comments_html:
+            f.write('### Комментарии\n\n' + post_comments_html)
```

Comment:
Another user has requested this string to be translatable, have a look at the implementation in #12